plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -95,6 +143,7 @@ def complete_version(self):
     def complete_version_serialized(self):
         return tuple(version_utils.encode(field) for field in self.complete_version)
 
+    # TODO DANGER DANGER, WHAT HAPPENS WITH MongoEngine BaseDocument???
```

Comment:
I did not write this @barnabycourt did. I'm not familiar with what the implications could be for this, but I've considered trying to answer that question by putting an interactive debugger statement here and either (a) running pulp smash and/or (b) do hand testing of functionality we expect to work. I'm suggesting the debugging approach because using grep to identify areas of usage, or some other simple static analysis, would be hard and likely incomplete.

What do you think?